rewrite-java/src/main/java/org/openrewrite/java/ChangePackage.java

```diff
@@ -118,7 +118,7 @@ public JavaSourceFile visitJavaSourceFile(JavaSourceFile cu, ExecutionContext ct
                 )));
 
                 for (J.Import anImport : c.getImports()) {
-                    if (anImport.getPackageName().equals(changingTo)) {
+                    if (anImport.getPackageName().equals(changingTo) && !anImport.isStatic()) {
```

Comment:
This fix affects both static imports of enums as static imports of methods, which is a good thing I believe.